### PR TITLE
Add a config option to only show the country code in store switcher

### PR DESCRIPTION
**Pull request by kolaente**

### CODE REVIEW

File: etc/adminhtml/system.xml [hunk 1]
```diff
@@ -11,10 +11,15 @@
             <resource>IMI_StoreSwitch::config</resource>
             <group id="general" translate="label" type="text" sortOrder="10" showInDefault="1" showInStore="1" showInWebsite="1">
                 <label>General</label>
-                <field id="enable" translate="label" type="select" sortOrder="10" showInDefault="1" showInStore="1" showInWebsite="1">
+                <field id="enable" translate="label" type="select" sortOrder="1" showInDefault="1" showInStore="1" showInWebsite="1">
```
Comment (DanieliMi) [inline etc/adminhtml/system.xml]:
```suggestion
                <field id="enable" translate="label" type="select" sortOrder="1" showInDefault="1" showInStore="1" showInWebsite="1" canRestore="1">
```

Comment (DanieliMi) [reply etc/adminhtml/system.xml]: To be able to reset to default.

File: etc/adminhtml/system.xml [hunk 2]
```diff
@@ -11,10 +11,15 @@
...
                     <label>Enable Switch</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="show_country_only" translate="label" type="select" sortOrder="2" showInDefault="1" showInStore="1" showInWebsite="1">
```
Comment (DanieliMi) [inline etc/adminhtml/system.xml]:
```suggestion
                <field id="show_country_only" translate="label" type="select" sortOrder="2" showInDefault="1" showInStore="1" showInWebsite="1" canRestore="1">
```

**Review by DanieliMi — CHANGES_REQUESTED** [review]
I've made some suggestions. I like when you have the option to restore default values so it would be nice to implement it.

**Comment by DanieliMi** [comment]
Maybe add to the readme how it looks when `show_country_only` is enabled and disabled.

File: etc/adminhtml/system.xml (hunk 2 above)
Comment (kolaente) [reply etc/adminhtml/system.xml]: Makes sense, thanks!

**Review by DanieliMi — APPROVED** [review]
Looks good. Thanks!

**Comment by kolaente** [comment]
@DanieliMi I've added screenshots, you can merge now.

**Review by DanieliMi — APPROVED** [review]

**Pull request merged**